api-client/src/api/mockClient.ts

```diff
@@ -1,15 +1,16 @@
 import * as dotenv from "dotenv"
 import * as jsonServer from "json-server"
 import data from "../mockData"
+import routes from "../routes.json"
 
 export function mockApp() {
     dotenv.config()
     const server = jsonServer.create()
-
-    server.use(jsonServer.rewriter({
-        '/api/v1/': '/$1',
-        '/api/v1/quiz/:slug': '/quiz?slug=:slug'
-    }))
+    const rules = {}
```

Comment:
Should this be `let` instead of `const` as the loop is reassigning `rules`

Reply:
I think it is assigning a property inside of the object defined.
Also lint pops up an error whenever we use let instead of const :)